src/casp/ml/data/distributed_anndata_dataset.py

```diff
@@ -0,0 +1,26 @@
+import typing as t
+
+from torch import Tensor
+from torch.utils.data import Dataset
+
+from . import DistributedAnnCollection
+
+
+class DistributedAnnCollectionDataset(Dataset):
+    def __init__(self, dac: DistributedAnnCollection) -> None:
+        self.dac = dac
+
+    def __len__(self) -> int:
+        return len(self.dac)
+
+    def __getitem__(self, index: int) -> t.Tuple[Tensor, int]:
+        """
+        :return: Tuple of tensor with a cell gene counts and db index.
+        """
+
+        # TODO: review with yerdos
+        v = self.dac[index]
```

Comment:
@ordabayevy @fedorgrab -- this is the datatype that was returned in for the original PCA model, just want to verify that this is what we want here.